src/mapblock_mesh.cpp

```diff
@@ -130,18 +130,8 @@ static u8 getInteriorLight(enum LightBank bank, MapNode n, s32 increment,
 	const NodeDefManager *ndef)
 {
 	u8 light = n.getLight(bank, ndef);
-
-	while(increment > 0)
-	{
-		light = undiminish_light(light);
-		--increment;
-	}
-	while(increment < 0)
-	{
-		light = diminish_light(light);
-		++increment;
-	}
-
+	if (light)
```

Comment:
imo it would be cleaner to write this as `light != 0` as we are using it as a number not a boolean.

Reply:
Or `light > 0`; it’s unsigned so that’s all the same.